### PR TITLE
Cross posting

### PR DESCRIPTION
**Pull request by talhasch**
@feruzm commits include support for cross post reading. Could you help me testing while i am preparing cross post sending tool.

### CODE REVIEW

File: src/common/i18n/locales/en-US.json [hunk 1]
```diff
@@ -203,7 +205,10 @@
     "show-history": "Show edit history",
     "hidden-warning": "This post was hidden due to low ratings.",
     "address-copy": "Copy post address",
-    "address-copied": "Post address copied to clipboard."
+    "address-copied": "Post address copied to clipboard.",
+    "browse-original": "Browse to the original post by @{{n}}",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:209]: `Browse original discussion` or `Open original post`, I like first one, what do you think?

Comment (talhasch) [reply src/common/i18n/locales/en-US.json:209]: "Open original post" makes sounds more appropriate .

Comment (feruzm) [reply src/common/i18n/locales/en-US.json:209]: ok, let's choose that and after main category fix, we can merge this PR 👍

Comment (talhasch) [reply src/common/i18n/locales/en-US.json:209]: hold on

**Comment by talhasch** [comment]
@feruzm done

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**